### PR TITLE
Add Cypress e2e tests

### PR DESCRIPTION
**Pull request by hankolsen**
Add Cypress for creating end-to-end tests.
Add tests for:
- creating a note
- marking a note as favorite
- deleting a note

As described in #73

### CODE REVIEW

**Comment by taniarascia** [comment]
Excellent, thank you for setting this up! Will work on reviewing it soon.

File: cypress/integration/note.spec.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    cy.get('.note-list-each.active .note-options').click()
+    cy.get('[data-cy="note-option-favorite-button"]').click()
+    cy.get('[data-cy="note-option-favorite-button"]').should('contain', 'Remove Favorite')
+    cy.get('[data-cy="favorites"').click()
+    cy.get('.note-list')
+      .children()
+      .should('have.length', 1)
+  })
+
+  it('shoukld send a note to trash', () => {
```
Comment (Jfelix61) [inline cypress/integration/note.spec.ts]: There is a typo on 'should'.

File: package.json [hunk 2]
```diff
@@ -10,7 +10,8 @@
     "eject": "react-scripts eject",
     "postinstall": "patch-package",
     "start": "react-scripts start",
-    "test": "react-scripts test"
+    "test": "react-scripts test",
+    "e2e": "npx cypress open"
```
Comment (MartinRosenberg) [inline package.json]: This can just be `"cypress open"`

**Comment by MartinRosenberg** [comment]
@all-contributors Please add @hankolsen for tests

**Pull request merged**